### ISSUE TITLE
TypeError: entriesDeep.flat is not a function

### ISSUE THREAD

**Issue by aylubimov** [acrazing/dpdm#3]
Trying to run dpdm on a TypeScript project, I get the following error:

> TypeError: entriesDeep.flat is not a function
>    at Object.<anonymous> (C:\Tools\nvm\v10.16.3\node_modules\dpdm\lib\bin\dpdm.js:81:26)
>    at step (C:\Tools\nvm\v10.16.3\node_modules\dpdm\node_modules\tslib\tslib.js:139:27)
>    at Object.next (C:\Tools\nvm\v10.16.3\node_modules\dpdm\node_modules\tslib\tslib.js:120:57)
>    at fulfilled (C:\Tools\nvm\v10.16.3\node_modules\dpdm\node_modules\tslib\tslib.js:110:62)

**Comment by marcioferlan** [comment]
Same issue here in a JavaScript project...

```
$ dpdm --tree --circular src/index.js 
TypeError: entriesDeep.flat is not a function
    at Object.<anonymous> (/Users/developer/.nvm/versions/node/v10.15.3/lib/node_modules/dpdm/lib/bin/dpdm.js:81:26)
```

**Comment by acrazing** [comment]
It depends on node `v12`, I will fix this problem asap.

**Comment by acrazing** [comment]
Fixed in `3.2.3`, thanks.
